src/core/tUnification.ml

```diff
@@ -32,6 +32,7 @@ type eq_kind =
 
 type unification_context = {
 	allow_transitive_cast : bool;
+	allow_abstract_cast   : bool;
```

Comment:
Please, add a comment with the meaning of this flag. It's not immediately obvious.

Reply:
It allows a non-transitive abstract cast (`to`/`from`/`@:to`/`@:from`). Is there a better name?

Reply:
Just add that as a comment :)